app/src/components/map/SearchFeaturePopup.tsx

```diff
@@ -1,24 +1,20 @@
 import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
 import React from 'react';
 import { Popup } from 'react-leaflet';
 
-export const SearchFeaturePopup: React.FC<{ featureData: any }> = (props) => {
-  const { featureData } = props;
-
-  const popupItems = [];
-  for (const property in featureData) {
-    if (property !== 'geometry') {
-      popupItems.push(`${property}: ${featureData[property]}`);
-    }
-  }
+export const SearchFeaturePopup: React.FC<{ featureData: any; history: any }> = (props) => {
+  const { featureData, history } = props;
```

Comment:
Why does history have to be passed in as a prop? Can we not just have `const history = useHistory();` here directly?